README.md

```diff
@@ -858,11 +858,11 @@ a += [5, 6, 7, 8]
 
 #### 💡 Explanation:
 
-*  a += b doesn't behave the same way as a = a + b
+*  `a += b` doesn't always behave the same way as `a = a + b`.  Classes *may* implement the op= operators differently, and lists do this.
```

Comment:
@cben Thanks for the PR. Nice catch! Just a minor question, shouldn't "op=" be syntax highlighted as `op=`, or at least it'd be better to wrap it around quotes, I feel.

Reply:
done.  `op=` is not actual syntax but discovered I can make *`op=`* italic, now I'm happy :-)